### ISSUE TITLE
open up AC hole

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/dankdryer#1]
the side hole for our ac cabling is too small. i think it's currently 5mm. 7mm is probably better. we might also want to see about moving it to the fan side, or putting a TPU gasket in there -- right now it's on a hotpath for air, and we'll likely lose some out of it.

**Comment by dankamongmen** [comment]
A naive enlargement shows an obstruction on the top. What could this be?
![2024-09-17-214457_942x534_scrot](https://github.com/user-attachments/assets/a210cd4d-b8ae-4682-90e2-c2f9f52422c9)

**Comment by dankamongmen** [comment]
yeah it starts around -72.3 whatever size we make the hole

**Comment by dankamongmen** [comment]
are we maybe improperly tilted?

**Comment by dankamongmen** [comment]
lengthening the cylinder eliminates this. that makes sense if we're shorter than the wall thickness, but then wouldn't we have seen it at 5?

we don't have an explicit calculation for wall thickness on the croom. i guess it would be `(0.05 * totalxy) / 2`? let's see what that is...5.325. and we were 10, so yeah, i don't think that makes sense. oh, our cylinder could be misplaced, though, yeah that's surely it. let's fix it the right way, then.

**Comment by dankamongmen** [comment]
you know what? i think we're improperly titled, as well...

![2024-09-17-230652_1113x546_scrot](https://github.com/user-attachments/assets/f33d96dd-5066-4ffc-9559-9912ef439df4)

**Comment by dankamongmen** [comment]
holy shit, i've been misunderstanding the effects of `scale()` this entire time. we're calculating different wall thicknesses at the bottom and top because *the walls have different thickness at bottom and top*, with the wall being thinner at the top. think of it: scaling to 95% everywhere doesn't hold the wall thickness constant, dumbass. but we've been thinking it would for whatever reason.

alright, rather than using `scale()`, use `hull()` and two `polygon()`s. update everything that works off 0.95 multipliers. use the same radii to the outer walls that we have now:

```c
toprad = outerxy / 2;
botrad = totalxy * sqrt(2) / 2;
```

with a `$fn=4` instance of `cylinder()`, then a polygon at `0.95 * botrad`, then a polygon at `toprad - 0.05 * botrad`, with a `hull()` over it. actually, no---`botrad` is distinct from `botrad - croomz`, which is where we want the recess to start. ok, the right triangle drops an altitude from the `toprad`. that hits an outer corner via `sqrt(2) * (0.05 * botrad)`. oh wait we can do this more easily: `-flr / croomz * sqrt(2) * (0.05 * botrad)`

ok i'm just goin to redefine this more sensibly:
```
totalxy = spoold + wallxy * 2 + gapxy * 2; // hotbox altitude * 2. unchanged.
outerxy = (totalxy + 14) * sqrt(2); // hotbox radius. unchanged to reduce invalidations, though arbitrary
toprad = outerxy / 2; // unchanged
botrad = 
```

**Comment by dankamongmen** [comment]
ok, i've enlarged it to 7mm diameter, and placed it so that there is no obstruction. but it's longer than it ought be. reduce it when we fix #13.
